recipe/meta.yaml

```diff
@@ -1,39 +1,45 @@
-{% set version = '0.9' %}
-{% set sha256 = 'f5368329adbef86accd3bc9490522340bb79260455ae89b1a42c10f63801b9a6' %}
+{% set name = "hurry.filesize" %}
+{% set version = "0.9" %}
 
 package:
-  name: hurry.filesize
+  name: {{ name|lower }}
```

Comment:
As an aside, this is a pessimization in my eyes. Yes, the bot autogenerates this format, but it gives the false impression that the name is a variable that could be changed, and has zero benefits, only downsides (e.g. it makes it harder to copy-paste the correct URL). If any thing, you should be replacing `{{ name|lower }}` when you encounter one in a recipe, but at the very least please don't introduce it where it doesn't exist.

Reply:
@h-vetinari 
Did not know that - Grayskull generates with name: {{ name|lower }}, and that's always prompted making the change - and I thought I was making things better / more standard by replacing with the variable. !

Reply:
People can have different opinions which is better or worse, and though mine is pretty clear, it's certainly not the end of the world. The name variable is IMO an artefact of the automatisation approach (it's already a variable in grayskull, so the impulse to reuse it is there by default), but I haven't had the time to go fix it at the source. Probably I should one of these days...

Reply:
Had started to in PR: https://github.com/conda/grayskull/pull/548

Probably needs some tweaks before it can be merged